docs/public/static/styles/prism-okaidia.css

```diff
@@ -1,68 +1,88 @@
 /**
- * https://unpkg.com/prismjs/themes/prism-okaidia.css
- *
  * okaidia theme for JavaScript, CSS and HTML
  * Loosely based on Monokai textmate theme by http://www.monokai.nl/
  * @author ocodia
  */
 
-/* inline code */
-code {
-  padding: 0.1em;
-  border-radius: 0.3em;
-  white-space: normal;
-}
-
-code[class*='language-'] {
+code[class*='language-'],
+pre[class*='language-'] {
```

Comment:
I have synced the file with https://unpkg.com/prismjs/themes/prism-okaidia.css and commented what is not necessary.